migration/planning-migration-3-to-4.adoc

```diff
@@ -0,0 +1,158 @@
+
+For more information, see xref:../operators/olm-what-operators-are.adoc#olm-what-operators-are[Understanding Operators].
+
+[id="migration-differences-install"]
+=== Installation and update differences
+
+[discrete]
+==== Installation process
+
+To install {product-title} 3.11, you prepared your Red Hat Enterprise Linux (RHEL) hosts, set all of the configuration values your cluster needed, and then ran an Ansible playbook to install and set up your cluster.
+
+In {product-title} 4.2, you use the OpenShift installation program to create a minimum set of resources required for a cluster. Once the cluster is running, you use Operators to further configure your cluster and to install new services.
+
+For more information, see xref:../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process].
+
+If you want to add RHEL workers to your {product-title} 4.2 cluster, you use an Ansible playbook to join the RHEL workers after the cluster is running. For more information, see xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an OpenShift Container Platform cluster].
+
+[discrete]
+==== Infrastructure options
+
+In {product-title} 3.11, you installed your cluster on infrastructure that you prepared and maintained. In addition to providing your own infrastructure, {product-title} 4 offers an option to deploy a cluster on infrastructure that the {product-title} installation program provisions and the cluster maintains.
+
+For more information, see xref:../architecture/architecture-installation.adoc#installation-overview_architecture-installation[OpenShift Container Platform installation overview].
+
+[discrete]
+==== Upgrading your cluster
+
+In {product-title} 3.11, you upgraded your cluster by running Ansible playbooks. In {product-title} 4.2, The cluster manages its own updates, including updates to Red Hat Enterprise Linux CoreOS (RHCOS) on cluster nodes. You can easily upgrade your cluster by using the web console or by using the `oc adm upgrade` command from the OpenShift CLI and the Operators will automatically upgrade themselves. If your {product-title} 4.2 cluster has Red Hat Enterprise Linux workers, then you will still need to run an Ansible playbook to upgrade those workers.
+
+For more information, see xref:../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[Updating clusters].
+
+[id="migration-considerations"]
+== Migration considerations
+
+Review the changes and other considerations that might affect your migration from {product-title} 3.11 to {product-title} 4.
+
+[id="migration-preparing-storage"]
+=== Storage considerations
+
+Review the storage changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== TODO
```

Comment:
@gnufied Can you please let me know what storage items we need to mention to customers moving from OCP 3.11 to 4.2? Thanks!